tools/tidy

```diff
@@ -69,6 +75,11 @@ fi
 
 find-files() {
     local files=()
+
+    if [[ -f $filename ]]; then
+        ls $filename
```

Comment:
do you need to list the file instead of just `echo $filename`?

Reply:
I’m a bit paranoid, either way should be ok.

Reply:
The parameter should be quoted and considering the `if` it makes more sense to use `echo`.

Reply:
Actually I think the whole function `find_files` should only be called if no file parameter is supplied, e.g. change line R102 

```diff
- find-files | xargs perltidy --pro=.../.perltidyrc
+ [[ -f $filename ]] && perltidy --pro=.../.perltidyrc $filename || find-files | xargs perltidy --pro=.../.perltidyrc

Reply:
@okurz I like that! done.